source/reference/search-indexes.txt

```diff
@@ -33,17 +33,21 @@ second argument.
 
 .. code-block:: ruby
 
-  client[:bands].search_indexes.create_one({ dynamic: true })
-
+  # Creates indexes on all dynamically indexable fields with a default index name
   client[:bands].search_indexes.create_one(
-    {
-      dynamic: false,
-      fields: {
+    { mappings: { dynamic: true } }
+  )
+
+  # Creates an index on the specified field with the specified index name
+  index_definition = {
+    mappings: {
+      dynamic: false,  
+      fields: { 
         name: { type: 'string', analyzer: 'lucene.simple' }
```

Comment:
S: consider changing the name of the indexed field as `name` is also the name of an option to set the index name
```suggestion
        country: { type: 'string', analyzer: 'lucene.simple' }
```